configs/README.md

```diff
@@ -15,7 +15,7 @@ A detailed description of the parameters for model configuration (i.e., initiali
 | forcing_resolution | double (scalar)| - | sec/min/hr | temporal resolution | - | timestep of the forcing data. Recommended value of 3600 seconds. |
 | endtime | double (scalar)| >0 | sec, min, hr, d | simulation duration | - | time at which model simulation ends |
 | layer_soil_type | int (1D array) | - | - | state variable | - | layer soil type (read from the database file soil_params_file) |
-| max_soil_types | int | >1 | - | - | - | maximum number of soil types read from the file soil_params_file (default is set to 15) |
+| max_valid_soil_types | int | >1 | - | - | - | maximum number of valid soil types read from the file soil_params_file (default is set to 12; model not valid for soil_type = waterbody, lava, glacier, etc.) |
```

Comment:
It would be nice if this didnt require a name change if possible? Downstream tools will need to be updated to work with the new name and if avoidable, that would be preferred.

Reply:
that's a good point, however, I think a change at this stage won't hurt much as LGAR is in the early stages, later it will become a bit hard. Also, I am not aware if anyone is using it right now other than Peter and myself. What do you think?

Reply:
It is a little more clear, lets go with the change!